images/manageiq-ui-worker/Dockerfile

```diff
@@ -6,10 +6,3 @@ MAINTAINER ManageIQ https://manageiq.org
 
 LABEL name="manageiq-ui-worker" \
       summary="ManageIQ user interface worker image"
-
-RUN source /etc/default/evm && \
-    /usr/bin/generate_rpm_manifest.sh && \
-    dnf clean all && \
-    rm -rf /var/cache/dnf
-
-RUN rm -rf /tmp/rpms /create_local_yum_repo.sh /etc/yum.repos.d/local_rpm.repo
```

Comment:
I wasn't sure why this was done here.  Removing these doesn't change anything with respect to the file size, and one can get at them from one of the parent images.

Reply:
I think we used to install something here, but it was moved up in the layers.